packages/teuchos/parameterlist/test/ParameterList/ParameterList_UnitTests.cpp

```diff
@@ -1208,6 +1208,53 @@ TEUCHOS_UNIT_TEST( ParameterList, print ) {
   }
 }
 
+// define enum class Shape in anonymous namespace outside of unittest
+// "NonPrintableParameterEntries" to avoid polution of class type in string comparison
+namespace {
+    enum class Shape : int { CIRCLE, SQUARE, TRIANGLE };
+}
```

Comment:
This is an anonymous namespace from inside of the Teuchos namespace.  I did not realize the C++ allowed such things (but I guess that is harmless).